src/components/keyboard.tsx

```diff
@@ -13,21 +14,27 @@ const audioController = new AudioContextWrapper();
 
 function Keyboard() {
     const [isClicked, setIsClicked] = useState(false);
-    const octave = 4;
+    const [octave, setOctave] = useState(4);
```

Comment:
I think this is good for now, but I want to discuss architecture of the controller. I think it'd be solid to have an abstracted component responsible for all of these state objects, and with this component responsible solely for render the components and passing them the requisite info for them to get the information they need.